base_access_control_group.sol

```diff
@@ -17,12 +17,13 @@ BsAccessCtrlGrp20190722161600ML: Made editable
 BsAccessCtrlGrp20190722214400ML: Provides the list of members and managers
 BsAccessCtrlGrp20190723130500ML: Fixes typo in managersNum
 BsAccessCtrlGrp20190723165900ML: Fixes deletion/adding to groups
+BsAccessCtrlGrp20200529110800ML: Overloads checkRights to reflect difference between groups and wallets
```

Comment:
Add `BsAccessCtrlGrp20200303165900PO : add oauth state to groups.` before this line?